xcube/webapi/ows/stac/context.py

```diff
@@ -33,6 +33,15 @@ def __init__(self, server_ctx: Context):
         self._datasets_ctx = server_ctx.get_api_ctx('datasets',
                                                     cls=DatasetsContext)
 
+        # TODO: decide what else to include in this list.
+        #  Listing all 11396 CRSs currently known to pyproj seems
+        #  impractical. Make it a configuration option, perhaps?
```

Comment:
Is this really a TODO? If so, say why this is needed.

Actually I don't understand why we limit it to geographic. xcube can reproject to any CRS, see tiles generation.

Reply:
I've downgraded it to a "check".